snow/networking/tracker/resource_tracker_test.go

```diff
@@ -66,28 +68,20 @@ func TestCPUTracker(t *testing.T) {
 
 	node1Utilization := cpuTracker.Usage(node1, endTime2)
 	node2Utilization := cpuTracker.Usage(node2, endTime2)
-	if node1Utilization >= node2Utilization {
-		t.Fatalf("Utilization should have been higher for the more recent spender")
-	}
+	require.Greater(node2Utilization, node1Utilization)
 
 	cumulative := cpuTracker.TotalUsage()
 	sum := node1Utilization + node2Utilization
-	if cumulative != sum {
-		t.Fatalf("Cumulative utilization: %f should have been equal to the sum of the spenders: %f", cumulative, sum)
-	}
+	require.Equal(sum, cumulative)
 
 	mockUser.EXPECT().CPUUsage().Return(.5).Times(3)
 
 	startTime3 := endTime2
 	endTime3 := startTime3.Add(halflife)
 	newNode1Utilization := cpuTracker.Usage(node1, endTime3)
-	if newNode1Utilization >= node1Utilization {
-		t.Fatalf("node CPU utilization should decrease over time")
-	}
+	require.Greater(node1Utilization, newNode1Utilization)
```

Comment:
q: what would be use for stricly greater? I just want to make sure we are not making the test stricter

Reply:
This isn't making the test stricter, note the order of the variables compared to the original if statement.